library/ssl_tls13_keys.h

```diff
@@ -322,14 +322,11 @@ int mbedtls_ssl_tls1_3_derive_application_secrets(
  *
  * \param md_type           The hash algorithm used in the application for which
  *                          key material is being derived.
- * \param master_secret     The master secret from which the resumption master
```

Comment:
This should be `application_secret`, but the documentation should stay.

Reply:
Added back.